backend/src/main/resources/doc/openapi.json

```diff
@@ -1148,6 +1148,10 @@
                     },
                     "count": {
                         "type": "integer"
+                    },
+                    "lastConfirmedDate": {
```

Comment:
I suppose that you have edited it manually. This openApi documentation is exported from stoplight.io where should be edited, too.  It is placed here to generate DTO objects in this project. The truth between BE and FE is this OpenAPI document located in the loono-api project. The mobile app generates the API objects from this file on its side.

Reply:
Ok, is there a way I can get an access there?

Reply:
Bump :)

Reply:
@rposkocil done in https://github.com/cesko-digital/loono-api/pull/2